flake8_fine_pytest/watchers/usefixtures.py

```diff
@@ -14,24 +20,62 @@ def run(self) -> None:
         if self._should_run():
             self._validate_usefixtures_used_where_possible(self.tree)
 
-    def _validate_usefixtures_used_where_possible(self, tree: ast.AST) -> None:
-        for node in ast.walk(tree):
-            if not self._should_check_node(node):
-                continue
+    def _should_run(self) -> bool:
+        return self.options.force_usefixtures and super()._should_run()
```

Comment:
I understand correctly that we have to do such a check for all settings? How about to make a class attribute and take the validation to the base class via `getattr`?

Reply:
not really, there's either an enable/disable option for a validator, or a integer/list options which defines validator's behavior.
So, in general that's not an should_run/should_not_run flag.